src/css/orders.css

```
@@ -0,0 +1,75 @@
+#all-orders{
+  width:100%;
+  height:100%;
+  display:flex;
+
+}
+
+#orders, #delivered-orders{
+width:45%; 
+margin:3rem;
+height:100%;
+}
+
+.header-delivered, .header-order{
+  color:#f1e9cd;
+  text-align:center;
+  font-size:2rem;
+  margin-bottom:0.3rem;
+}
+
+#orders-container, #delivered-orders-container{
+  border: 7px double #f1e9cd;
+  width:100%;
+  display:flex;
+  justify-content:center;
+  flex-wrap: wrap;
+}
```

Add a min-height here
```suggestion
#orders-container, #delivered-orders-container{
  border: 7px double #f1e9cd;
  width:100%;
  display:flex;
  justify-content:center;
  flex-wrap: wrap;
  min-height: 150px;
}
```